mesonbuild/compilers/rust.py

```diff
@@ -245,6 +275,26 @@ def get_assert_args(self, disable: bool, env: 'Environment') -> T.List[str]:
         action = "no" if disable else "yes"
         return ['-C', f'debug-assertions={action}', '-C', 'overflow-checks=no']
 
+    def get_rust_tool(self, name: str, env: 'Environment') -> T.List[str]:
```

Comment:
No need to quote annotations since we use `from __future__ import annotations`. It's not really worth the churn of rewriting tons of older code (that can be done when modifying the code anyway) but when writing new code please omit the quotes.